pkgs/by-name/mo/monaspace/fonts.toml

```
@@ -0,0 +1,38 @@
+version = "1.301"
+baseUrl = "https://github.com/githubnext/monaspace/releases/download"
+fontFamily = "monaspace"
+
+[[fonts]]
+hash = "sha256-H6J4InGyXabZuslywuzNYqw14zymzF90JKxa7CikOIM="
+variant = "static"
+destination = "opentype"
+
+[[fonts]]
+hash = "sha256-7O0qRA12ELPNs4919aW5rUdLs/XaLTJNthL+4Gjlx90="
+variant = "frozen"
+destination = "truetype"
+
+[[fonts]]
+hash = "sha256-cPAQBgrxIFXRjBRK9kvt3uYwewi5Lw/ryiwjUeVJWD4="
+variant = "variable"
+destination = "truetype"
+
+[[fonts]]
+hash = "sha256-okhtInxOcIjS3A2w2q6f4EloMF7I8nBuD5aSreQr9bI="
```

This is the hash that I get when I try to build it. Please verify.
```suggestion
hash = "sha256-3/JOe+aSSA2vxnKHjSwkxukqNYMajsIuQ68IJvY07XI="
```